configs/components/augeas.rb

```diff
@@ -1,12 +1,40 @@
 component 'augeas' do |pkg, settings, platform|
-  pkg.version '1.8.1'
-  pkg.md5sum '623ff89d71a42fab9263365145efdbfa'
-  pkg.url "#{settings[:buildsources_url]}/augeas-#{pkg.get_version}.tar.gz"
+  # Projects may define an :augeas_version setting, or we use 1.8.1 by default:
+  version = settings[:augeas_version] || '1.8.1'
+  pkg.version version
 
-  if platform.is_sles? && platform.os_version == '10'
-    pkg.apply_patch 'resources/patches/augeas/augeas-1.2.0-fix-services-sles10.patch'
```

Comment:
SLES 10 is no longer supported in puppet-agent, and pdk doesn't need it

Reply:
can you move this into a separate commit?